lib/src/default_revset_graph_iterator.rs

```diff
@@ -37,7 +37,7 @@ use crate::revset::{RevsetGraphEdge, RevsetGraphEdgeType};
 // |/          ~
```

Comment:
Perhaps change this comment to `///` so that it is more discoverable? I think that would make it appear in VS Code's popup.

Reply:
Makes sense. Done with this one and the other comments further down.